### ISSUE TITLE
ReferenceError: COLOR_WHEEL is not defined

### ISSUE THREAD

**Issue by gerhardcit** [c0bra/color-scheme-js#10]
Install and run on node 6.9 
npm install color-scheme

```
let scheme = new ColorScheme();
        scheme.from_hex(mainColor)
            .scheme('triade')
            .distance(0.7)
            .variation('pastel');
```
Error:
`ReferenceError: COLOR_WHEEL is not defined`

**Comment by orouz** [comment]
+1

**Comment by c0bra** [comment]
Should be fixed with v0.0.7 (just published). Please let me know if it's still an issue.

**Comment by c0bra** [comment]
Actually, just tested this locally and it looks fine so I'll close.
